Modules/SubModule_Structural/xASL_wrp_GetVolumetrics.m

```diff
@@ -95,22 +95,26 @@ function xASL_wrp_GetVolumetrics(x)
     xASL_adm_DeleteFileList(x.D.TissueVolumeDir, ['WMH_LST_(LGA|LPA)_' x.P.SubjectID '.(csv|tsv)'], [], [0 Inf]);
     
     % Run the LST lesion volume calculator
-	% -> Here, we change the current directory to the SUBJECT DIRECTORY as ps_LST_tlv outputs to the current directory
+	% -> Here, we change the current directory to the SUBJECT DIRECTORY as ps_LST_tlv outputs to the current (SUBJECT) directory
     CurrDir = pwd;
     cd(x.dir.SUBJECTDIR);
     [~, FileName] = ps_LST_tlv(x.P.Path_WMH_SEGM, ~bVerbose, BinThresh, MinimalLesionVolume);
     cd(CurrDir);
     
-    % get current filename:
+    % Get the current filename in the SUBJECT DIRECTORY
     FileName = fullfile(x.dir.SUBJECTDIR, FileName);
-    % define new filename:
-    OutputPath = fullfile(x.dir.SUBJECTDIR, ['WMH_LST_' x.WMHsegmAlg '_' x.P.SubjectID '.csv']);
+    % Define new filenames:
+    OutputPath = fullfile(x.D.TissueVolumeDir, ['WMH_LST_' x.WMHsegmAlg '_' x.P.SubjectID '.csv']);
     % Then move our file
     if ~exist(FileName, 'file')
          warning(['Missing:' FileName]);
 	else
+		% First rename the output file to the correct filename format in the population dir
```

Comment:
If you remove the word "format" here, I'm fine :)
Because the format is changed below.

And I like that you make the comments this explicit! ;)

Reply:
that was to convince you :)
Sure "format" was a typo ;)

Reply:
I'll do this once I'm done with running other tests. So please only check the function above.